.github/actions/artifact_download/action.yml

```
@@ -0,0 +1,30 @@
+name: Download artifact
+description: Download and decrypt an artifact.
+inputs:
+  name:
+    description: 'The name of the artifact.'
+    required: true
+  path:
+    description: 'Download to a specified path.'
+    required: false
+    default: ./
+  encryption-secret:
+    description: 'The secret to use for decrypting the artifact.'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install unzip
+      uses: montudor/action-zip@v1
+    - name: Create temporary directory
+      id: tempdir
+      shell: bash
+      run: echo "directory=$(mktemp -d)" >> "$GITHUB_OUTPUT"
+    - name: Download the artifact
```

```suggestion
    - name: Create temporary directory
      id: tempdir
      shell: bash
      run: echo "directory=$(mktemp -d)" >> "$GITHUB_OUTPUT"
      
    - name: Download the artifact
```
empty lines to separate steps